script/Arcueid/Day 9/09_00_ARC09_1_3.txt

```diff
@@ -13,7 +13,7 @@ If you think about it calmly, it sounds absurd.
 -- 不自然に転任してきた担任……自称：美人教師……が、実は吸血鬼から街を守る修道服のお姉さんだった、とか、唐突にも程がある。
 // Noel calls herself 'the onee-san of justice who protects the city', so preserved onee-san here too
 The homeroom teacher who transferred out of nowhere...
- the self-proclaimed 'beautiful teacher'...
+ The self-proclaimed 'beautiful teacher'...
```

Comment:
Feel like this is all the same sentence, including the "is, in reality" on the next line which is still uncapitalized here